### PR TITLE
Add Avalonia to the .NET Open Source Developer Projects A cross-platform UI framework that supports XAML and .NET

### PR DESCRIPTION
**Pull request by Apollo9999**
Added Avalonia to the .NET Open Source Developer Projects A cross-platform UI framework that supports XAML and .NET

### CODE REVIEW

**Review by drraghavendra — COMMENTED** [review]
changes are made

**Review by richlander — APPROVED** [review]

File: dotnet-free-oss-services.md [hunk 1]
```diff
@@ -43,8 +44,12 @@ Please sort projects alphabetically and provide a one-line description. Where po
 ## Analytics and Error Reporting
 
 * [Bugsnag](https://bugsnag.com/blog/bugsnag-loves-open-source) - Cross platform error reporting.
+* [Countly](https://count.ly/
```
Comment (mairaw) [inline dotnet-free-oss-services.md:47]:
```suggestion
* [Countly](https://count.ly/)
```

File: dotnet-free-oss-services.md [hunk 2]
```diff
@@ -43,8 +44,12 @@ Please sort projects alphabetically and provide a one-line description. Where po
...
+* [Raygun](https://raygun.com/blog/tags/open-source/) - Cross platform error reporting and Application Performance Monitoring.
```
Comment (mairaw) [inline dotnet-free-oss-services.md:48]:
```suggestion
* [Raygun](https://raygun.com/blog/tags/open-source/) - Cross-platform error reporting and application performance monitoring.
```

File: dotnet-free-oss-services.md [hunk 3]
```diff
@@ -43,8 +44,12 @@ Please sort projects alphabetically and provide a one-line description. Where po
...
 * [Sentry](https://sentry.io/for/open-source/) - Cross platform error reporting.
 
+
+
```
Comment (mairaw) [inline dotnet-free-oss-services.md:52]:
```suggestion
```

**Comment by mairaw** [comment]
@Apollo9999 can you reply with the appropriate CLA answer if you agree with it?

File: dotnet-free-oss-services.md [hunk 4]
```diff
@@ -43,6 +44,8 @@ Please sort projects alphabetically and provide a one-line description. Where po
 ## Analytics and Error Reporting
 
 * [Bugsnag](https://bugsnag.com/blog/bugsnag-loves-open-source) - Cross platform error reporting.
+* [Countly](https://count.ly/)
```
Comment (richlander) [inline dotnet-free-oss-services.md:47]: Can you add a description here?

**Comment by Apollo9999** [comment]
git config user.kiran
git config user.tech.correspondent@electronicsmaker.com
git commit --author="<tech.correspondent@electronicsmaker.com>"

**Comment by Apollo9999** [comment]
Please approve the Pull Request

**Comment by Apollo9999** [comment]
@microsoft-github-policy-service agree

**Comment by Apollo9999** [comment]
Respected @mairaw please look into the above PR

**Comment by EssaAlKazem** [comment]
releases/net481/README.md

File: dotnet-free-oss-services.md [hunk 5]
```diff
@@ -43,8 +44,7 @@ Please sort projects alphabetically and provide a one-line description. Where po
 ## Analytics and Error Reporting
 
 * [Bugsnag](https://bugsnag.com/blog/bugsnag-loves-open-source) - Cross platform error reporting.
-* [elmah.io](https://elmah.io/sponsorship/opensource/) - Error logging and uptime monitoring for .NET.
-* [Sentry](https://sentry.io/for/open-source/) - Cross platform error reporting.
```
Comment (terrajobst) [inline dotnet-free-oss-services.md:47]: Why the deletion?

**Review by terrajobst — CHANGES_REQUESTED** [review]
I'd like to understand the deletion before approving

**Comment by Apollo9999** [comment]
Sir @terrajobst please respond

**Review by terrajobst — APPROVED** [review]

**Pull request merged**